minio/operator
# Adding kes-configuration example on tenant helm chart

# How to test

- deploy vault on your cluster following this documentation https://github.com/minio/operator/blob/master/docs/examples.md#minio-tenant-with-encryption-enabled-using-vault-kms
- install operator
- uncomment kes configuration and set vault credentials on helm/tenant/values.yaml
- install tenant via `helm install --namespace tenant-ns --create-namespace tenant ./helm/tenant`
- MinIO tenant should be deployed with `KES` enabled
- verify encryption is working fine with `mc admin kms key status alias --insecure`

Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>
